src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -315,8 +319,9 @@ def _testcase_reproduces_in_revision(testcase,
   _log_output(revision, result)
 
   if update_metadata:
-    progression_task_output.issue_metadata = _get_and_update_issue_metadata(
-        testcase)
+    progression_task_output.ClearField("issue_metadata")
```

Comment:
Can you call `clear()` on the map field instead of clearing the field from the proto entirely?

```suggestion
    progression_task_output.issue_metadata.clear()
```

It avoids one string reference to a static field and expresses intent more precisely anyway.